src/ansible_runner/config/runner.py

```diff
@@ -158,14 +158,20 @@ def prepare(self):
     def prepare_inventory(self):
         """
         Prepares the inventory default under ``private_data_dir`` if it's not overridden by the constructor.
+
+        We make sure that if inventory is a path, that it is an absolute path.
         """
         if self.containerized:
             self.inventory = '/runner/inventory'
             return
 
         if self.inventory is None:
+            # At this point we expect self.private_data_dir to be an absolute path
+            # since that is expanded in the base class.
             if os.path.exists(os.path.join(self.private_data_dir, "inventory")):
                 self.inventory = os.path.join(self.private_data_dir, "inventory")
+        elif isinstance(self.inventory, str) and os.path.exists(self.inventory):
```

Comment:
Looks good to me. @Shrews Do you think we should update https://ansible.readthedocs.io/projects/runner/en/latest/standalone/#specifying-an-alternate-inventory stating this change?

Reply:
I'm not sure what you're suggesting we should point out in documentation. We aren't changing any functionality here, just fixing a bug.